src/Common/src/System/Net/ContextAwareResult.cs

```diff
@@ -153,7 +153,7 @@ internal ExecutionContext ContextCopy
                 ExecutionContext context = _context;
                 if (context != null)
                 {
-                    return context.CreateCopy();
+                    return context; // No need to copy on CoreCLR; ExecutionContext is immutable
```

Comment:
Ah, right.  I know we'd discussed adding CreateCopy to the contract, but I forgot that we never actually did it. :)